examples/sdk-app-example/src/main/res/layout/main.xml

```diff
@@ -35,7 +35,6 @@
       style="@style/ButtonTheme"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:onClick="crashWithNativeSignal"
```

Comment:
this is redundant as a click listener is set programmatically within the mainactivity